pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.autopep8]
+
+max_line_length = 80
+in-place = true
+aggressive = 3
+# Add the path to here if you want to exclude them from autopep8 auto reformat.
+exclude = '''
```

Comment:
The following folders should also be excluded:
- apps, benchmarks, cmake, conda, docker, docs, featgraph, third_party, tutorials

The following folders can be omitted for now but should be checked later:
- tests

Reply:
Added part of it. It is not clear to me why python code in apps/ should be ignored.
If we feel it is important to add, we can do it later.

Reply:
They are legacy codes that shall we removed.
* dgl-ke has been moved to https://github.com/awslabs/dgl-ke
* dgl-lifesci has been moved to https://github.com/awslabs/dgl-lifesci